### DISCUSSION TITLE
Getters in devtools

### DISCUSSION THREAD

**Discussion by marshallswain** [vuejs/pinia#435] [Ideas]
I find that being able to see the getters in the devtools is really valuable for debugging.  There's not really a performance downside on the apps that I build, but would rather discuss before making a PR.

The cleanest, most-maintainable way I can see would be to add an array of the getter property names to the root level of the store here: https://github.com/posva/pinia/blob/v2/src/store.ts#L226

Then a third group of fields can be added to the `formatStoreForInspectorState` function, here: https://github.com/posva/pinia/blob/v2/src/devtools.ts#L169. I did a quick change to `pinia.esm-browser.js` in `node_modules` with the following change (Note: I should have set `editable` to `false` for getters):
![Screen Shot 2021-04-09 at 5 20 41 PM](https://user-images.githubusercontent.com/128857/114249466-ec43c180-9957-11eb-9f23-e5f3f832f772.jpg)

The result works as expected:
![Screen Shot 2021-04-09 at 5 23 11 PM](https://user-images.githubusercontent.com/128857/114249596-5197b280-9958-11eb-9943-c376812507ec.jpg)

@posva, if you are open to this approach, I will create a PR.  Thanks for building `pinia`.  The API is nice and clean and easy to use.

**Comment by posva** [comment]
I'm glad you are enjoying Pinia! 😄 

I think we can add an internal property to the store only in dev mode for the getters `_getters: (string | symbol)[]` that can be _reduced_ in the devtools.ts file. We should probably do something similar for actions but if you want to give just the `_getters` a try, go ahead!

**Reply by marshallswain** [reply]
OK.  I'll submit a PR.  Thank you!

**Reply by posva** [reply]
I implemented this at c4bf761

**Reply by marshallswain** [reply]
Ooh thanks!  That was on my list for today.
